src/server.rs

```diff
@@ -398,7 +392,7 @@ impl<S, M> Handler for Server<S, M> where S: Store, M: StateMachine {
 
 impl <S, M> fmt::Debug for Server<S, M> where S: Store, M: StateMachine {
     fn fmt(&self, fmt: &mut fmt::Formatter) -> fmt::Result {
-        write!(fmt, "Server({})", self.id)
+        fmt::Debug::fmt(&self.id, fmt)
```

Comment:
Indeed. Considering the number is that of the ServerId, I thought that it may be suitable while the Server may be interpreted as the token or slot/position/counter. The ServerId is also constant to the peer, correct?

I did comment on this for discussion however, that was on the previous version of this commit (pre wrapped_enum update). Forgot to re-add it.